Client/Frontend/Theme/Theme.swift

```diff
@@ -34,6 +34,8 @@ protocol ThemeColourPalette {
     var layerAccentPrivate: UIColor { get }
     var layerAccentPrivateNonOpaque: UIColor { get }
     var layerLightGrey30: UIColor { get }
+    // Only for Reader mode Sepia background
+    var layerSepia: UIColor { get }
```

Comment:
` // Only for Reader mode Sepia background`
I don't think it's necessary to add this comment since this is more of a designer concern. Like what colors they need to use in the application.